tf/start.sh

```diff
@@ -88,7 +88,7 @@ do
     for netarch in ${NETARCHS[@]}
     do
       echo "Training $netarch:"
-      train "$CONFIGDIR/$netarch.yaml" "${netarch}-$(date +"%Y_%m%d_%H%M_%S_%3N").txt"
+      train "$CONFIGDIR/$netarch.yaml" "${netarch}-$(date +"%Y_%m%d_%H%M_%S_%3N")"
```

Comment:
train() needs updating since train.py now outputs .pb.gz for weights - but its going to try and gzip a non existant file and then try to move that result to net dir.